locale/ru/loader_modernized.cfg

```diff
@@ -1,81 +1,83 @@
+fast-mdrn-loader-split=Используется для погрузки/выгрузки предметов со скоростью конвейера.
+express-mdrn-loader-split=Используется для погрузки/выгрузки предметов со скоростью конвейера.
+turbo-mdrn-loader-split=Используется для погрузки/выгрузки предметов со скоростью конвейера.
+ultra-fast-mdrn-loader=Используется для погрузки/выгрузки предметов со скоростью конвейера.
+extreme-fast-mdrn-loader=Используется для погрузки/выгрузки предметов со скоростью конвейера.
+ultra-express-mdrn-loader=Используется для погрузки/выгрузки предметов со скоростью конвейера.
+extreme-express-mdrn-loader=Используется для погрузки/выгрузки предметов со скоростью конвейера.
+original-ultimate-mdrn-loader=Используется для погрузки/выгрузки предметов со скоростью конвейера.
+ultra-fast-mdrn-loader-split=Используется для погрузки/выгрузки предметов со скоростью конвейера.
+extreme-fast-mdrn-loader-split=Используется для погрузки/выгрузки предметов со скоростью конвейера.
+ultra-express-mdrn-loader-split=Используется для погрузки/выгрузки предметов со скоростью конвейера.
+extreme-express-mdrn-loader-split=Используется для погрузки/выгрузки предметов со скоростью конвейера.
+original-ultimate-mdrn-loader-split=Используется для погрузки/выгрузки предметов со скоростью конвейера.
+
+[technology-name]
+mdrn-loader=Погрузчик
+fast-mdrn-loader=Быстрый погрузчик
+express-mdrn-loader=Экспресс-погрузчик
+turbo-mdrn-loader=Турбо-погрузчик
+
+[technology-description]
+mdrn-loader=Разблокируйте быструю погрузку предметов с конвейера или на конвейер.
+fast-mdrn-loader=Разблокируйте быструю погрузку предметов с конвейера или на конвейер.
+express-mdrn-loader=Разблокируйте быструю погрузку предметов с конвейера или на конвейер.
+turbo-mdrn-loader=Разблокируйте быструю погрузку предметов с конвейера или на конвейер.
+
+[mod-setting-name]
+mdrn-use-electricity=Использовать электричество
+mdrn-double-recipe=Двойной рецепт
+mdrn-migrate-from-miniloaders=Миграция с мини-погрузчиков
+mdrn-enable-chute=Включить скат
+mdrn-enable-stacking=Включить складывание в стопки
+mdrn-use-aai-graphics=Использовать графику AAI Loaders
+
+[mod-setting-description]
+mdrn-use-electricity=Если включено, погрузчики требуют электропитание для работы.
+mdrn-double-recipe=Удваивает стоимость рецепта и производит 2 погрузчика одновременно, чтобы подземные конвейеры расходовались парами.
+mdrn-migrate-from-miniloaders=Попытка перенести построенные мини-погрузчики в Loaders Modernized. Погрузчики в инвентаре и контейнерах будут потеряны. Рецепты сборщиков будут сброшены.
+mdrn-enable-chute=Включить рецепт, чтобы создавать очень медленный скат в стиле погрузчика.
+mdrn-enable-stacking=Включить складывание предметов в стопки на конвейере, согласно уровню технологии складывания.
+mdrn-use-aai-grphics=Обнаружен мод AAI Loaders.\nИспользуется только графика из AAI Loaders для Loaders Modernized.\n\nЕсли вы установите значение "false" и перезапустите, вы сможете использовать и Loaders Modernized, и AAI Loaders.

```

Comment:
I had a typo in my original file - sorry.  Should be 'mdrn-use-aai-graphics'

Reply:
Hello.
I don't see any changes to the English locale to change the translation accordingly.
What's the problem with the CRLF? It's always been fine in all translations.

Reply:
Nothing wrong with the CRLF from a functional, technical point of view.  I've built the original files as UNIX/Mac line ending and switching them to DOS base line endings causes every line to be marked as changed in the merge.  It's a style thing and wanting to keep the commits not pulling in line-ending changes.